datadog_checks_dev/README.md

```diff
@@ -551,6 +551,21 @@ Options:
   -h, --help         Show this message and exit.
 ```
 
+#### Validate
+
+```console
+$ ddev validate -h                                                                                                                                                                                                                                                                      [16:56:07]
```

Comment:
What is the date for?

Reply:
Also instead of `ddev validate` could you change it to `ddev metadata verify`. This is because we already have `ddev manifest verify` and so i just want it to be consistent